Docs/source/acknowledge_us.rst

```diff
@@ -14,7 +30,8 @@ Please acknowledge the role that WarpX played in your research by adding the fol
   \usepackage{hyperref}
   This research used the open-source particle-in-cell code WarpX \url{https://github.com/ECP-WarpX/WarpX}, primarily funded by the US DOE Exascale Computing Project. We acknowledge all WarpX contributors.
 
-**Main WarpX reference:**
+Main WarpX reference
+********************
 
 .. raw:: html
```

Comment:
If you like, we recently updated the DOI in the readme to the latest Publication, because it contains an up-to-date, C++y description of the code base: https://doi.org/10.1063/5.0028512

Reply:
Is this the one we should cite from now on?

Reply:
Yep, just confirmed again via Slack with @jlvay.

Reply:
sounds good, thanks, I updated it.